tests/BUILD.bazel

```diff
@@ -10,6 +10,15 @@ py_test(
     ],
 )
 
+py_test(
+    name = "test_get_package_version",
+    timeout = "short",
+    srcs = ["test_get_package_version.py"],
+    deps = [
+        "//src:dependency-bazelizer",
```

Comment:
you don't need the entire dependency-bazelizer, src:get_package_version should be enough

Reply:
thx